apiserver/pkg/model/converter.go

```diff
@@ -236,12 +236,10 @@ func PopulateServeConfig(serveConfigSpecs []v1alpha1.ServeConfigSpec) []*api.Ser
 
 func PoplulateRayServiceStatus(serviceName string, serviceStatus v1alpha1.RayServiceStatuses, events []v1.Event) *api.RayServiceStatus {
 	status := &api.RayServiceStatus{
```

Comment:
@kevin85421 Could you shed some light on what this is used for? IIRC in one of our discussions you mentioned you disabled the api server by default?

Reply:
I did not know anything about KubeRay API Server. If you want to update the API Server in this PR, I can ping some API Server folks to review that part. We can also revert the changes about API Server, and ask the API Server folks for help.